code/modules/reagents/Chemistry-Goon-Medicine.dm

```diff
@@ -0,0 +1,335 @@
+#define SOLID 1
+#define LIQUID 2
+#define GAS 3
+
+#define REM REAGENTS_EFFECT_MULTIPLIER
+
+datum/reagent/silver_sulfadiazine
+	name = "Silver Sulfadiazine"
+	id = "silver_sulfadiazine"
+	description = "100% chance per cycle of healing 2 points of BURN damage."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+	metabolization_rate = 2
+
+datum/reagent/silver_sulfadiazine/reaction_mob(var/mob/living/M as mob, var/method=TOUCH, var/volume)
+	if(method == TOUCH)
+		M.adjustFireLoss((volume-(volume*2))*REM)
+		M << "You feel your burns healing!"
+		M.emote("scream")
+	if(method == INGEST)
+		M.adjustToxLoss(0.5*volume)
+		M << "You probably shouldn't have eaten that. Maybe you should of splashed it on, or applied a patch?"
+	..()
+	return
+
+datum/reagent/styptic_powder
+	name = "Styptic Powder"
+	id = "styptic_powder"
+	description = "100% chance per cycle of healing 2 points of BRUTE damage."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+	metabolization_rate = 2
+
+datum/reagent/styptic_powder/reaction_mob(var/mob/living/M as mob, var/method=TOUCH, var/volume)
+	if(method == TOUCH)
+		M.heal_organ_damage(volume*REM,0)
+		M << "You feel your wounds knitting back together!"
+		M.emote("scream")
+	if(method == INGEST)
+		M.adjustToxLoss(0.5*volume)
+		M << "You probably shouldn't have eaten that. Maybe you should of splashed it on, or applied a patch?"
+	..()
+	return
+
+datum/reagent/salglu_solution
+	name = "Saline-Glucose Solution"
+	id = "salglu_solution"
+	description = "100% chance per cycle of healing 1 point each of OXY and TOX damage."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+
+datum/reagent/salglu_solution/on_mob_life(var/mob/living/M as mob)
+	if(M.stat == DEAD)
+		return
+	if(!M) M = holder.my_atom
+	if(M.getOxyLoss()) M.adjustOxyLoss(-1*REM)
+	if(M.getToxLoss()) M.adjustToxLoss(-1*REM)
+	..()
+	return
+
+datum/reagent/synthflesh
+	name = "Synthflesh"
+	id = "synthflesh"
+	description = "100% chance per cycle of healing 1 point each of BRUTE and BURN damage."
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+
+datum/reagent/synthflesh/reaction_mob(var/mob/living/M, var/method=TOUCH, var/volume)
+	if(!M) M = holder.my_atom
+	if(method == TOUCH)
+		M.heal_organ_damage(volume*REM,0)
+		M.adjustFireLoss((volume-(volume*2)*REM))
```

Comment:
I'd like to add that synthflehs is meant to be heal 1.5 brute and fire both per unit applied and not 1; it also stops bleeding (though I realize that isn't a thing on TG and is reserved purely to Bay and Goon).